gossip/src/cluster_info.rs

```diff
@@ -1217,6 +1232,27 @@ impl ClusterInfo {
             .collect()
     }
 
+    pub fn get_restart_last_voted_fork_slots(
+        &self,
+        cursor: &mut Cursor,
+    ) -> Vec<RestartLastVotedForkSlots> {
+        let self_shred_version = self.my_shred_version();
+        let gossip_crds = self.gossip.crds.read().unwrap();
+        gossip_crds
+            .get_entries(cursor)
+            .filter_map(|entry| match &entry.value.data {
+                CrdsData::RestartLastVotedForkSlots(slots) => {
+                    if slots.shred_version == self_shred_version {
+                        Some(slots.clone())
+                    } else {
+                        None
+                    }
+                }
+                _ => None,
+            })
```

Comment:
This would be probably more readable as:
```rust
.filter_map(|entry| {
    let CrdsData::RestartLastVotedForkSlots(slots) = &entry.value.data else {
        return None;
    };
    (slots.shred_version == self_shred_version).then_some(slots)
})
.cloned()
.collect()
```

Reply:
Done.